charts/kong/ci/test4-values.yaml

```diff
@@ -34,3 +34,12 @@ dblessConfig:
             message: "dbless-config"
 proxy:
   type: NodePort
+# - add stream listens
```

Comment:
Why are these added to two DB-less deployments? Can we move one over to a DB deployment?
It would also be best if we can keep one with IC and one without.

Total 2 combinations.

Reply:
Because I cannot read comments good :(